README.md

```diff
@@ -35,6 +39,9 @@ Properties
 ----------
 - **credentials**: UID and PWD used for database authentication
 - **database**: name of database to connect to
+- **enrich**: enable signal enrichment
+- **id**: 
```

Comment:
needs to be filled out. on a side note, this is a silly thing to have to fill out.